kof-operator/internal/controller/isito/remote_secret.go

```diff
@@ -0,0 +1,456 @@
+package istio
```

Comment:
Please keep "Copyright Istio Authors" header as that is just a copy-paste from istio repo